cli/src/main/scala/com/codacy/analysis/cli/configuration/Configuration.scala

```diff
@@ -0,0 +1,11 @@
+package com.codacy.analysis.cli.configuration
+
+import better.files.File
+import com.codacy.analysis.cli.command.Properties
+
+object Configuration {
+
+  def baseProjectDirectory(directory: Option[File]): File =
```

Comment:
We need to avoid doing this type of changes in the middle of other changes.

Still I think this would be better inside the environment class.